### ISSUE TITLE
Learning Rate Annealing

### ISSUE THREAD

**Issue by remzr7** [rbrigden/a3c#1]
Write code to implement the learning rate annealing as described in the a3c paper. 

- [x] Figure out if this is doable with the default keras optimizer

- [ ] If so great. If not we may need to go the nuclear option and manually build gradient flow and updates into the system ourselves

**Comment by rbrigden** [comment]
Updates on this?

**Comment by rbrigden** [comment]
@tonyf pointed out that annealing is part of shared RMSProp as detailed in the paper. Closing this.
